resources/views/layouts/navigation.blade.php

```diff
@@ -16,6 +16,11 @@
                         {{ __('Dashboard') }}
                     </x-nav-link>
                 </div>
+                <div class="hidden space-x-8 sm:-my-px sm:ml-10 sm:flex">
+                    <x-nav-link :href="route('wishlist')" :active="request()->routeIs('wishlist')">
+                        {{ __('WishList') }}
+                    </x-nav-link>
+                </div>
```

Comment:
Instead of linking in the nav bar, we could move it over to the dropdown. I tried this around line 40 and it looked alright in the view. Let me know what you think!

                        <!-- Authentication -->
                        <form method="POST" action="{{ route('logout') }}">
                            @csrf

                            <x-dropdown-link :href="route('wishlist')" 
                                    :active="request()->routeIs('wishlist')">
                                {{ __('Wishlist') }}
                            </x-dropdown-link>

                            <x-dropdown-link :href="route('logout')"
                                    onclick="event.preventDefault();
                                                this.closest('form').submit();">
                                {{ __('Log Out') }}
                            </x-dropdown-link>

Reply:
The only thing to point out here is to make sure we aren't including extra dropdown-links inside of the logout form

Reply:
That part is totally on me, it should outside the form method, will update